src/main/java/com/gtnewhorizons/angelica/mixins/Mixins.java

```diff
@@ -29,6 +28,10 @@ public enum Mixins {
             ,"angelica.MixinEntityRenderer"
             ,"angelica.MixinMinecraft"
             ,"angelica.optimizations.MixinRendererLivingEntity"
+            ,"angelica.dynamiclights.MixinEntityRenderer"
```

Comment:
Might be worth making this into a module-config to entirely enable/disable it like the other modules

Reply:
We could, default on, but default mode is OFF

Reply:
Yah I'm fine with the module defaulting to on, just want to give users the option to entirely disable it if it conflicts with another mod they're using.  I'm even fine having the default option to be on 🤷